README.md

```diff
@@ -38,9 +38,9 @@ In your Maven Java project, add the plugin to your `pom.xml`:
 
 ### Configuration
 
-Configure the plugin by changing `registry`, `repository`, and `credentialHelperName` accordingly.
+Configure the plugin by changing `registry`, `repository`, and `credentialHelperName` in the registry of your choice. For example:
 
-#### I am using Google Container Registry (GCR)
+Using Google Container Registry (GCR)...
```

Comment:
Should this still be in header4?

Reply:
I prefer to minimize the use of headers, but we can keep them if you prefer. Up to you.

Reply:
I like headers mostly because they can be easily linked to, like https://github.com/google/jib#i-am-using-google-container-registry-gcr

Reply:
Ok, that makes sense.
But, I'm not crazy about leading with i am. 
Are you ok with :
### Using Google Container Registry (GCR)

Reply:
Yep, that looks good!